### DISCUSSION TITLE
next js with redux saga

### DISCUSSION THREAD

**Discussion by ujjwal11** [vercel/next.js#17154] [Help]
i m using next js latest version 9.5.3 with the redux(saga) i created some dispatch method using saga but now when i m calling my dispatch method it says :- TypeError: Cannot read property 'dispatch' of undefined
this is my page

import React, {Component} from 'react'
import Footer from "../src/containers/Footer/footerView"
import Header from "../src/containers/Header/headerView"
import HomeView from "../src/containers/HomeContent/homeView"
import {connect} from 'react-redux'
import {loadHomePageData} from '../src/containers/HomeContent/actions'
import {loadDataMegaNav} from '../src/containers/MegaNavContent/actions'

const Home = (props) => {
  return (
   <p>first page</p>
   
  )
}
export async function getServerSideProps(context) {
  console.log('contextcontext>>>',context)
  const { store, isServer } = context

  store.dispatch(loadHomePageData())

  return { props: { store : 'ssss' } }
}

export default connect(state=>state)(Home)

**Comment by stegano** [comment]
Refer to the link below 
- https://github.com/kirill-konshin/next-redux-wrapper#getserversideprops
